src/prefect/cli/flow_run.py

```diff
@@ -279,3 +279,33 @@ async def execute(
         exit_with_error("Could not determine the ID of the flow run to execute.")
 
     await Runner().execute_flow_run(id)
+
+
+@flow_run_app.command()
+async def resume(id: UUID):
+    """Resume a flow run by ID."""
+    async with get_client() as client:
+        try:
+            resume_state = State(
```

Comment:
Hi @JiginJayaprakash looks great! 
just one question, why not using the state convenience function [Scheduled](https://github.com/JiginJayaprakash/prefect/blob/a7eff1b948d7b29b00111fd4e65744878f123f39/src/prefect/states.py#L488)?

Reply:
Hi, 
thank you. I have used the state because it was present in the Cancel command and thought it was the way to update state

If you want ,I can update it